frontend/pages/admin/TeamManagementPage/TeamTableConfig.tsx

```diff
@@ -77,8 +77,8 @@ const generateTableHeaders = (
       ),
     },
     {
-      title: "Members",
-      Header: "Members",
+      title: "Users",
```

Comment:
@rachaelshaw @noahtalerman 
Note: Not outlined on Figma, I also changed the title of the Teams table Members column to Users.
<img width="1365" alt="Screenshot 2024-02-01 at 9 26 08 AM" src="https://github.com/fleetdm/fleet/assets/71795832/8bfad96f-3e1d-4b35-8f18-7adbf2339a30">